Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -743,6 +743,12 @@ ModelExperimental.prototype.update = function (frameState) {
 
   if (!Matrix4.equals(this.modelMatrix, this._modelMatrix)) {
     this._sceneGraph.updateModelMatrix(this);
+    this._modelMatrix = Matrix4.clone(this.modelMatrix);
```

Comment:
wait a sec... not sure why we had both `this.modelMatrix` and `this._modelMatrix`... is the underscore one used for anything except for checking if the matrix is dirty? I think this should be turned into a proper getter/setter and just use a dirty flag for consistency.

Reply:
The current design is intentional so that the model matrix can be updated in place without needing to use the setter. See https://github.com/CesiumGS/cesium/pull/9997.

Reply:
Maybe a comment is needed since it's the only property that deviates from the dirty flag approach